usagov/usagov-2021
# USAGOV-1226: Replace CKEditor4 with CKEditor5

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1226

## Description

Updates several ckeditor5-related drupal modules and updates the text editor configuration "USWDS" for CKEditor5.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Update your site: make sure to run `bin/build` **AND** `bin/drush cim`
- Add or edit a page. Make sure that the **Text format** (just under the editor field for the Body field) is set to **USWDS**. A good existing page to try is http://localhost/apply-adult-passport
- Verify that you can add and edit USWDS components. Click the paragraphs button (last button in the editor toolbar) to add a component. Double-click an existing component to edit it. 
- Verify that there is no unsynchronized configuration showing up here: http://localhost/admin/config/development/configuration
- The content team will do a thorough test on dev. 

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
